andela/mmdp_site
# MMDP-401 Prevent manual access of admin paths on ui

**What does this PR do?**

Protect admin paths from manual access.

**Description of Task to be completed?**

Currently, it is possible for users to access users and groups paths on the UI although without access to the API. Protect them from manual access by non-permitted users.

**How should this be manually tested?**

1. Fetch all branches - `git fetch --all`.
2. Checkout to this branch - `git checkout bg-fix-ui-admin-routes-access-MMDP-401`.
3. Run the application - `yarn dev-server`.
4. Log in with a basic account i.e. one without users and groups access.
5. Manually enter user and group paths to ascertain that it is impossible to access them. Attempts to access any of these paths should show the landing page. Below is the list of paths:
    - `/group`
    - `/group/edit/:id`
    - `/group/list`
    - `/users`
    - `/users/edit/:email/:username`
    - `/users/all`

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-401](https://viisaus.atlassian.net/browse/MMDP-401)

**Screenshots (if appropriate)**

![screenshot 2019-03-08 at 15 37 34](https://user-images.githubusercontent.com/8180548/54029117-6cffc480-41b8-11e9-85d9-d7cd8b5274c3.png)

**Questions:**

None
